springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/Message.java

```diff
@@ -35,4 +34,13 @@ public class Message {
     private PayloadReference payload;
 
     private HeaderReference headers;
+
+
+    @Override
+    public int compareTo(Message o) {
```

Comment:
Why? For sorting?

Reply:
The ChannelMerger#mergeOperation uses a Set to remove duplicate messages for the same channel.
The "same" message can be provided by multiple scanners, but with slightly different title, description, header information.

If you find it unsuitable here, I can also look into alternatives to detect "duplicates"

Reply:
In that case:
1. Wouldn't the equals and hashCode be enough? Why is Comparable required?
2. I think that in case of multiple messages in an operation, the differentiator should be the payload type, no?

Reply:
I had a closer look. What I described above is already achieved with the added `@EqualsAndHashCode(of = {"name"})`.

The field name is the canonical name of the class. With multiple messages, all messages with the same canonical name (=payload type) are merged together.

I added one more test to the ChannelMergerTest to confirm this. This also confirms, that the `compareTo` is not needed and therefore removed in the latest commit.